include/swift/ABI/Metadata.h

```diff
@@ -292,7 +293,7 @@ struct TypeLayout;
 /// the requirements of some specific type.  The information in
 /// a value-witness table is intended to be sufficient to lay out
 /// and manipulate values of an arbitrary type.
-struct ValueWitnessTable {
+template <typename Runtime> struct TargetValueWitnessTable {
```

Comment:
This doesn't really work: all of the field types are host types instead of target types.  You're actually skating by because you're testing a 64-bit target from a 64-bit host.